prover/Cargo.lock

```diff
@@ -8460,6 +8507,7 @@ dependencies = [
  "jsonrpsee",
  "pin-project-lite",
  "rlp",
+ "rustls",
  "serde",
  "serde_json",
  "thiserror",
```

Comment:
are changes to this file done on purpose?

Reply:
Yes; can elaborate on "why" if needed.